packages/chakra-ui-core/src/Radio/index.js

```diff
@@ -67,6 +67,9 @@ const Radio = {
         props: {
           as: 'input'
         },
+        domProps: {
```

Comment:
Good catch!

Remember to remove the `defaultIsChecked` prop from the `attrs` key as well.

Reply:
done